GNUmakefile

```diff
@@ -245,6 +245,10 @@ golangci-lint: ## Run golangci-lint for all issues.
 	[ -d "./golangci-lint" ] || mkdir ./golangci-lint && \
 	docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v2.5.0 golangci-lint run --max-issues-per-linter 0 --max-same-issues 0 | tee ./golangci-lint/$(shell date +%Y-%m-%d_%H:%M:%S).txt
 
+.PHONY: lint-fix
+lint-fix: gomods ## Run golangci-lint with --fix for all modules
+	gomods -u -go run github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.5.0 run --fix
```

Comment:
Should we define v2.5.0 as a global var in the Makefile to keep `lint-fix` and `lint` targets in sync?